packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/ContentTypeBuilderNav.tsx

```diff
@@ -53,7 +50,7 @@ export const ContentTypeBuilderNav = () => {
                 defaultMessage: section.title.defaultMessage,
               })}
               collapsable
-              badgeLabel={section.links.length.toString()}
+              badgeLabel={section.linksCount.toString()}
             >
```

Comment:
Completely up to you but we could remove the hook changes and do this. I think
it achieves the same thing?

```suggestion
              badgeLabel={section.links
                .reduce((acc, link) => acc + (link.links ? link.links.length : 1), 0)
                .toString()}
```

Reply:
Yes it does the same so a matter of personal preference. I think I prefer extending the hook like I did because there was already a `if (hasChild) {` condition, so I can put the logic in the same place